### ISSUE TITLE
Support downloading timeline photos posted by other users

### ISSUE THREAD

**Issue by ash211** [mohsen1/fb-photo-fetch#13]
Currently this script downloads photo from the album named "Timeline Photos" that's returned from listing the user's albums.  That album contains photos the user posted to their wall ("feed").  But there are additional photos relevant to the user on their wall ("feed") that are not posted by the user, but instead by the user's friends.  If those photos have a face tag applied, then that photo appears in the "Photos of me (Tagged photos)" folder this script generates.  But if the user is not face tagged on these photos, they cannot be retrieved.

In summary, we currently cannot download photos that:
- are posted on the user's wall
- by another user
- that do not have a facetag of the user

The problem here is that the FB Graph API for the feed [1] claims to support showing posts the user was tagged in, but the functionality is actually broken.  See stackoverflow [2] and the confirmed bug [3] for details.

Once, if ever, FB supports that API endpoint, we can extend this fetcher project to also download those photos.  Until then, we are sadly incomplete in photo fetching from FB.

[1] https://developers.facebook.com/docs/graph-api/reference/v2.12/user/feed
[2] https://stackoverflow.com/questions/38670358/facebook-graph-api-getting-all-posts-a-user-is-tagged-in-even-if-the-privacy-se
[3] https://developers.facebook.com/bugs/284611378567364

**Comment by mohsen1** [comment]
interesting! I don't have photos like that but I assume some people have their photos "sent" to them like this. 

I'm kind of pessimistic about this as well. FB is tightening their APIs more than ever now. At this point someone should use an Electron app to scrape FB instead of using their APIs to get _their own_ photos!

**Comment by ash211** [comment]
Many thanks for your work on this repo by the way!  Best project I found to get my photos out of FB without losing the dates.

I have a good number of photos like this (only 2 out of a few hundred on my timeline were uploaded by me) so am hopeful FB opens up the API, but agreed it's not looking hopeful.

The electron app approach would be something like having a user open a group of photos (e.g. the theater) and then they hit something to have the electron app scan through and download the photos + fix exif?

**Comment by mohsen1** [comment]
Yes, an automated browser. Maybe [puppeteer](https://github.com/GoogleChrome/puppeteer) is a better candidate actually.
